lib/libzutil/os/linux/zutil_device_path_os.c

```diff
@@ -273,7 +273,7 @@ zfs_get_pci_slots_sys_path(const char *dev_name)
 			free(address2);
 			if (asprintf(&path, "/sys/bus/pci/slots/%s",
 			    ep->d_name) == -1) {
-				free(tmp);
+				free(path);
```

Comment:
This doesn't look right.  In the event `asprintf()` fails and we take this code path `path` will be undefined.

Reply:
I just pushed a revision that addresses this.